dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/registry/WorkerRegistry.java

```diff
@@ -152,7 +152,7 @@ public void unRegistry() {
             // trim and lower case is need
             workerZkPathBuilder.append(workGroup.trim().toLowerCase()).append(SLASH);
             workerZkPathBuilder.append(address);
-            workerZkPathBuilder.append(weight).append(SLASH);
+            workerZkPathBuilder.append(weight);
```

Comment:
@CalvinKirs Please check if remove this SLASH, will the weight load balance produce error?

Reply:
It is not very good to put in heartbeatinfo, maybe we should strip out all the metadata information, it might be better

Reply:
It is not very good to put in heartbeatinfo, maybe we should extract all the metadata information and report it to zk separately, which may be better

Reply:
@yangyichao-mango Yes, I also think it's better to separate them